File: competitive-events-api/src/main/java/es/jocasolo/competitiveeventsapi/dto/score/ScorePostDTO.java
package es.jocasolo.competitiveeventsapi.dto.score;

import java.io.Serializable;

import javax.validation.constraints.NotBlank;

import es.jocasolo.competitiveeventsapi.dto.DTO;
import lombok.Getter;
import lombok.Setter;

public class ScorePostDTO extends DTO implements Serializable {

	private static final long serialVersionUID = 1L;

	@Getter
	@Setter
	private String value;

	@Getter
	@Setter
	@NotBlank
	private String eventId;

	@Override
	public String toString() {
		return "ScorePostDTO [value=" + value + "]";
	}

}

File: competitive-events-app/app/src/main/java/es/jocasolo/competitiveeventsapp/dto/score/ScorePostDTO.kt
package es.jocasolo.competitiveeventsapp.dto.score

import com.google.gson.annotations.SerializedName
import es.jocasolo.competitiveeventsapp.dto.BackStackEntryDTO

class ScorePostDTO(
        @SerializedName("value") var value: String? = null,
        @SerializedName("eventId") var eventId: String? = null,
) : BackStackEntryDTO() {
}